task_01.rst

```diff
@@ -0,0 +1,110 @@
+#################
+Why I Love Python
+#################
+
+Why I love **Python**, a language worth knowing. 
+
+**************
+Running It
+**************
```

Comment:
Your header over/underlines should be the same length as your text, eg, instead of

```
#########
Hello
#########
```

Make it:

```
#####
Hello
#####
```